### PR TITLE
Hotfix 4.1.0.1

### PR DESCRIPTION
**Pull request by binaryoverload**
## Removed
 - Useless error in resolving videos

## Fixed
 - Handle AuditLog errors for banning
 - NPE in LockChatCommand
 - Handle concurrency in ExpiringMap
 - Permissions args errors

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java [hunk 1]
```diff
@@ -162,7 +162,6 @@ public static AudioItem resolveItem(Player player, String input) throws IllegalA
                 if (e.getMessage().contains("Vevo")) {
                     throw new IllegalStateException(Jsoup.clean(cause.getMessage(), Whitelist.none()), cause);
                 }
-                FlareBot.LOGGER.error(Markers.NO_ANNOUNCE, "Cannot get video '" + input + "'");
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java]: How come?

Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java]: This get's logged to sentry unnecessarily, we don't get any debug info from it anyway. All errors are logged to the users in the channel.

Chose to silence the errors like this because 9/10 times it is something to do with youtube or connection and so it's not worth logging the events.

Comment (WalshyDev) [reply src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java]: But you aren't silencing? The point of this and then Markers.NO_ANNOUNCE is to silence it, if you run Sentry through our Filter then it will also handle correctly with the NO_ANOUNCE. I'm against this change

Comment (binaryoverload) [reply src/main/java/stream/flarebot/flarebot/util/GeneralUtils.java]: Even if I did that, there is no point in logging the error itself, all it does it spam the logs with cannot get video with no debug info or anything like that.

File: src/main/java/stream/flarebot/flarebot/util/objects/expiringmap/ExpiringMap.java [hunk 2]
```diff
@@ -9,18 +9,20 @@
 import java.util.TreeMap;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
+import java.util.concurrent.ConcurrentNavigableMap;
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/util/objects/expiringmap/ExpiringMap.java]: Remove unused import

File: src/main/resources/logback.xml [hunk 3]
```diff
@@ -1,19 +1,18 @@
...
     <appender name="FILE" class="ch.qos.logback.core.FileAppender">
-        <filter class="stream.flarebot.flarebot.util.errorhandling.Filter" />
         <file>./latest.log</file>
         <encoder>
             <pattern>[%d{yyyy.MM.dd HH:mm}] [%t] [%level] %msg%n</pattern>
         </encoder>
     </appender>
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
         <filter class="stream.flarebot.flarebot.util.errorhandling.ErrorCatcher" />
-        <filter class="stream.flarebot.flarebot.util.errorhandling.Filter" />
```
Comment (WalshyDev) [inline src/main/resources/logback.xml]: Why did you remove the filter?

Comment (binaryoverload) [reply src/main/resources/logback.xml]: Because the same functionality exists in error catcher, would be a redundant filter
